### ISSUE TITLE
Inaccurate wording of FAQ answer

### ISSUE THREAD

**Issue by rachaelshaw** [trailsjs/trails#63]
Some of the wording in the answer to 'Is Trails a fork of Sails?' is a little misleading, so would be a good idea to change it to avoid confusion about this project. Right now it says:
> Trails is built and maintained by former members of the Sails.js core team

But that makes it sound like the core maintainers of Sails have become involved in this project. It would be more accurate to change it to something along the lines of:
> Trails is built and maintained by a former Sails contributor and his team

(Would have submitted a PR myself, but I figured you might want to work with the wording a bit more before changing it.)

**Comment by tjwebb** [comment]
> But that makes it sound like the core maintainers of Sails have become involved in this project.

Around this time last year, @mikermcneil invited us to maintain Sails and take over community operations. Some of the things we did:
- organized and addressed hundreds of github issues, both with documentation-level and code-level resolutions
- maintained and contributed effort to Sails, Waterline, Skipper, Anchor, and several database adapters.
- traveled around the country (as well as internationally) to market the Sails.js framework and teach people how to use it
- wrote documentation and blog posts on how to incorporate various 3rd party libraries with Sails
- tagged over a thousand of sails.js questions on StackOverflow
- onboarded new corporate clients to use Sails.js in their enterprise, and train their teams how to make better use of it
- funded development of over a dozen open-source addons (hooks, adapters, and other miscellaneous plugins) for Sails.js

We took @mikermcneil seriously when he said he wanted us to take over operations while he worked on his Treeline project. For most of the past year, we spent **a lot** of time and money doing these things, and oriented our entire business around how we can best support Sails in the course of our normal business.

With this in mind, I find it perfectly appropriate that we assert ourselves as "former members of the Sails.js core team".

**Comment by mikermcneil** [comment]
> We took @mikermcneil seriously when he specifically said he wanted us to take over operations while he worked on his Treeline project. We did exactly what he asked us to do.

I appreciate your past work evangelizing Sails, but unfortunately this statement is not true.  I certainly asked for your help in organizing the community, but I never asked you to "take over operations". From the beginning, I was clear that I would maintain complete control of Sails, and that I would be its BDFL.  I reiterated this every time you repeatedly asked me for more rights, such as NPM publish access to Sails, which I refused.

> I find it perfectly appropriate that we assert ourselves as "former members of the Sails.js core team". Since this assertion also happens to be true, I don't intend to change it.

The untrue part of that statement is the plural "members".  The only person who was removed from any of repos was you, Travis.  This was after you told @particlebanana you were done contributing and would be forking Sails and had started posting negative tweets about my character.

**Comment by tjwebb** [comment]
Holy flame war! If you want [yet](https://news.ycombinator.com/item?id=10819583)[1]  [another](https://news.ycombinator.com/item?id=10777087)[2]  [article](https://news.ycombinator.com/item?id=10755557)[3] about "is Sails dead?" to end up on HackerNews, please by all means, proceed to hash this out on Github and have it perennially enshrined in the foreverness of the internet.

> The only person who was removed from any of repos was you, Travis.

So you'd like me to remove the word "former", then? Yes yes, we all already know you kicked me out of the "balderdashy" github org. I'm basing my nomenclature decision on what contributions and work actually occurred, not how many internet accounts you happen to have the passwords to.

> had started posting negative tweets about my character.

Yes, it's true that I posted screenshots of twitter of things that actually happened. If these led you to a negative interpretation of your character, it might be worth it to reflect on why that might be. 

> but I never asked you to "take over operations"

I have a pile of emails and a Licensing Agreement *from you and your lawyers* that says otherwise. I really fail to understand why you're clinging to desperately to this storyline, not why you're continuing with it here in an issue on github.com.

**Comment by tjwebb** [comment]
TLDR, @rachaelshaw and @mikermcneil, I won't be merging this feature.

**Comment by mikermcneil** [comment]
> I have a pile of emails and a Licensing Agreement from you and your lawyers that say otherwise. I really fail to understand why you're clinging to desperately to this storyline, and why you're continuing with it here in an issue on github.com.

@tjwebb the agreement you're referring to was never signed, and had only to do with a limited license to use the Balderdash brand for client work, and had nothing to do with Sails.

**Comment by mikermcneil** [comment]
> So you'd like me to remove the word "former", then? Yes yes, we all already know you kicked me out of the "balderdashy" github org. I'm basing my nomenclature decision on what contributions and work actually occurred, not how many internet accounts you happen to have the passwords to.

No, I'd like you to stop claiming that other contributors were also removed from the Sails project.

**Comment by tjwebb** [comment]
> the agreement you're referring to was never signed

I know. After you sent it to us for signatures, you began avoiding us and eventually removed us from everything when we pressed the issue.

> No, I'd like you to stop claiming that other contributors were also removed from the Sails project.

I don't think that the README makes that claim anywhere. Either way, I don't actually give a fuck about your opinion.

**Comment by mikermcneil** [comment]
As @rachaelshaw pointed out, the README says:

> Trails is built and maintained by former members of the Sails.js core team

However, none of the other members of the Sails core team are involved.  So we're asking if you'll please correct that.

**Comment by tjwebb** [comment]
> However, none of the other members of the Sails core team are involved

Now you have an issue with my use of plural vs. singular nouns? Again, see https://github.com/trailsjs/trails/issues/63#issuecomment-168800660. I am not the only person who I consider a core contributor as part of our arrangement, and putting forth effort into those activities.

**Comment by mikermcneil** [comment]
> After you sent it to us for signatures, you began avoiding us and eventually removed us from everything when we pressed the issue.

I removed you from everything because you told us you were done contributing to Waterline and Sails, and were going to create and promote your own fork.

**Comment by tjwebb** [comment]
> because you told us you were done contributing to Waterline and Sails

I told @particlebanana that if you continued to ignore us and the community, we would be forced to fork the project in order to maintain it. Then, yes, you kicked us out of the organization and our email accounts, and continued to ignore us. I'm glad we agree.

**Comment by rachaelshaw** [comment]
You also told me in pretty clear language that you were done.

But clearly this isn't a productive discussion -- I didn't mean to upset you, I just wanted to point out that the README makes it sound like this is by the authors of Sails, and you probably don't mean for it to come across that way.
